src/Tizen.NUI/src/public/Window/GLWindow.cs

```diff
@@ -494,6 +494,50 @@ public void Destroy()
             this.Dispose();
         }
 
+        /// <summary>
+        /// Enumeration for rendering mode
+        /// This Enumeration is used to choose the rendering mode.
+        /// It has two options.
+        /// One of them is continuous mode. It is rendered continuously.
+        /// The other is on demand mode. It is rendered by application.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public enum GLWindowRenderingMode
```

Comment:
How about changing 'RenderingModeType'?

Reply:
Thank you for your kind review.
Actually, DALi has the similar enumeration. The name is "RenderingMode".
I think same name is better.